Extension/src/LanguageServer/extension.ts

```diff
@@ -47,12 +47,10 @@ let realActivationOccurred: boolean = false;
 let tempCommands: vscode.Disposable[] = [];
 let activatedPreviously: PersistentWorkspaceState<boolean>;
 let buildInfoCache: BuildInfo | undefined;
-const taskTypeStr: string = "shell";
-const taskSourceStr: string = "C/C++";
 const cppInstallVsixStr: string = 'C/C++: Install vsix -- ';
 let taskProvider: vscode.Disposable;
 let codeActionProvider: vscode.Disposable;
-const intelliSenseDisabledError: string = "Do not activate the extension when IntelliSense is disabled.";
+export const intelliSenseDisabledError: string = "Do not activate the extension when IntelliSense is disabled.";
```

Comment:
Is this a user facing error that needs to be localized?

Reply:
I am not sure. I had to use this error in cppBuildTaskProvider, so I exported it.

Reply:
I see that this error is thrown in function `realActivation()` and was never localized. May not be user facing then.